core/jvm/src/main/scala/zio/RuntimePlatformSpecific.scala

```diff
@@ -44,4 +45,17 @@ private[zio] trait RuntimePlatformSpecific {
 
   final val defaultSupervisor: Supervisor[Any] =
     Supervisor.none
+
+  def enableLoomBasedExecutor(implicit trace: Trace): ZLayer[Any, LoomNotAvailableException, Unit] =
+    ZLayer.suspend {
+      LoomSupport.newVirtualThreadPerTaskExecutor() match {
+        case None           => ZLayer.fail(LoomNotAvailableException("Loom API not available", null))
+        case Some(executor) => Runtime.setExecutor(Executor.fromJavaExecutor(executor))
+      }
+    }
+
+  def enableAutoBlockingExecutor(implicit trace: Trace): ZLayer[Any, Nothing, Unit] =
```

Comment:
The one applied last will be used since they bother override the same thing. Auto blocking is only implemented in the ZScheduler, not the Loom executor.

Reply:
Got it, thanks